src/Compatibility/Microsoft.DotNet.PackageValidation/ValidatePackage.cs

```diff
@@ -38,6 +38,11 @@ protected override void ExecuteCore()
                 Package baselinePackage = NupkgParser.CreatePackage(BaselinePackageTargetPath, runtimeGraph);
                 new BaselinePackageValidator(baselinePackage, NoWarn, null, RunApiCompat, logger).Validate(package);
             }
+
+            if (!Log.HasLoggedErrors)
+            {
+                Log.LogMessage(MessageImportance.High, Resources.SuccessfulPackageRun);
```

Comment:
Not super relevant to the PR, but I'm not sure we should be setting this message with high importance.

Reply:
Yeah I was going to say importance normal or low and it will show on the binlog.

Reply:
we wont see normal or low importance in vs or console. I think it will be nice to log some info for the user.

Reply:
IMHO I don't think the user cares much about seeing a log that says the validation ran successfully, that should be implicit with the build being successful.

Reply:
Agreed, the minimal output is super reduced, today for a packed app it is not more than 10 lines, so I don't think that a message like this (specially something saying that validation was successful) is high enough in importance to have it.